### ISSUE TITLE
[TASK] Add app names to hidden apps dialog on wishlist

### ISSUE THREAD

**Issue by MxtOUT** [IsThereAnyDeal/AugmentedSteam#1635]
### Task Description

Display the app name besides the app image in the dialog

### Motivation

It may be hard to make out the name of the app just from the image + it's easier to search for a specific app

**Comment by candela97** [comment]
Yeah, but we'll need to get the app info from elsewhere. If we're gonna use any sort of Steam API, might as well implement this feature the proper way: manually build the missing rows.

**Comment by candela97** [comment]
App hovers `https://store.steampowered.com/apphover/[appid]` are a pretty minimal page to get this info, but it seems like it doesn't work for banned apps. Next is querying the community page, but I'd rather not do that.
